pkg/proof/README.md

```diff
@@ -0,0 +1,201 @@
+# Proof
+
+This package enables proof queries for Celestia transactions and blobs.
+
+## Details
+
+The following section will cover transaction inclusion proofs and blob inclusion proofs.
+
+## Prerequisites
+
+To understand the proof mechanisms,
+a good understanding of [binary merkle proofs](https://celestiaorg.github.io/celestia-app/specs/data_structures.html#merkle-trees) and [namespace merkle proofs](https://celestiaorg.github.io/celestia-app/specs/data_structures.html#namespace-merkle-tree) is required.
```

Comment:
Why are these lines split onto multiple lines? They don't currently split on some known text width like 80 characters so proposal to just put them all on one line.

```suggestion
To understand the proof mechanisms, a good understanding of [binary merkle proofs](https://celestiaorg.github.io/celestia-app/specs/data_structures.html#merkle-trees) and [namespace merkle proofs](https://celestiaorg.github.io/celestia-app/specs/data_structures.html#namespace-merkle-tree) is required.
```

Reply:
I still don't understand why these are lines are split on arbitrary bounds